.gitignore

```diff
@@ -226,4 +226,4 @@ ModelManifest.xml
 
 # SPAM configuration
 .spam/
-*.sarif
\ No newline at end of file
+.sarif/
```

Comment:
>sarif [](http://example.com/codeflow?start=1&length=5)

The previous entry ignored newly added test files! Clearly not write. I think it was intended to ignore everything in the .sarif\ directory...